core/jazz_es-kinesis-log-streamer/package.json

```diff
@@ -10,9 +10,11 @@
   "scripts": {
     "test": "mocha"
   },
-  "dependencies": {    
+  "dependencies": {
     "unicode-byte-truncate": "*",
-    "request": "*"
+    "request": "*",
+    "uuid": "*",
+    "aws-sdk": "*"
```

Comment:
`aws-sdk` is already part of the lambda runtime so just add it as devDependencies

Reply:
Done